kotsadm/pkg/handlers/app.go

```diff
@@ -149,6 +150,11 @@ func responseAppFromApp(a *apptypes.App) (*ResponseApp, error) {
 		return nil, errors.Wrap(err, "failed to check if gitops is supported")
 	}
 
+	isIdentityServiceSupported, err := store.GetStore().IsIdentityServiceSupportedForVersion(a.ID, a.CurrentSequence)
```

Comment:
these functions sure get the app config a whole lot unnecessarily. not a fix for this pr though

Reply:
totally agree, we should fix it